src/status_im2/subs/root.cljs

```diff
@@ -82,17 +82,16 @@
 ;; it is about cellular network/ wifi network
 (reg-root-key-sub :network/type :network/type)
 
-;;profile
+;;my profile
 (reg-root-key-sub :my-profile/seed :my-profile/seed)
-(reg-root-key-sub :my-profile/advanced? :my-profile/advanced?)
-(reg-root-key-sub :my-profile/profile :my-profile/profile)
-
-;;multiaccount
-(reg-root-key-sub :multiaccounts/multiaccounts :multiaccounts/multiaccounts)
-(reg-root-key-sub :multiaccounts/login :multiaccounts/login)
-(reg-root-key-sub :multiaccount :multiaccount)
-(reg-root-key-sub :multiaccount/accounts :multiaccount/accounts)
-(reg-root-key-sub :get-recover-multiaccount :multiaccounts/recover)
+
+;;profiles
+(reg-root-key-sub :profile/profiles-overview :profile/profiles-overview)
+(reg-root-key-sub :profile/login :profile/login)
+;; we have some fields only in overview, would be cool to merge them in status-go
```

Comment:
;; we have some fields only in overview, would be cool to merge them in status-go

imo this comment seems like it will never be seen by anyone. can we instead make an two issues for this. One in Status-Go repo and the other in Status mobile linking the two?

Reply:
yes, for sure, thanks

Reply:
but i would still keep this comment, just for devs who will be wondering why subscription name is different

Reply:
okay but can you update the comment with a link to the github issue?

Reply:
yes, done https://github.com/status-im/status-mobile/issues/16422